Gemfile

```diff
@@ -1,3 +1,4 @@
 source 'https://rubygems.org' do
+  gem 'cocoapods', '~> 1.10.0'
```

Comment:
Why not use the latest version `~> 1.11.0`?

Reply:
```suggestion
  gem 'cocoapods', '~> 1.11.0'
```